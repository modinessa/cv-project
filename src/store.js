import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
