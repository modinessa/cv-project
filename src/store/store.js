import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});