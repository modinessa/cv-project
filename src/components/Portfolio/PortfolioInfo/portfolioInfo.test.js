import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../../store/store';
import { PortfolioInfo } from './index';

const TEST_TITLE = 'Title';
const TEST_TEXT = 'Content bla-bla';
const TEST_URL = 'www.url.com'

describe('PortfolioInfo', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<PortfolioInfo
					title={TEST_TITLE}
					text={TEST_TEXT}
					url={TEST_URL}
				/>
			</Provider>
		)).toMatchSnapshot();
	})
})