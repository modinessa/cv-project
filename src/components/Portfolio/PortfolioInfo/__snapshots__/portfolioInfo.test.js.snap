// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PortfolioInfo renders component 1`] = `
<div
  className="card-info"
>
  <h4>
    Title
  </h4>
  <p>
    Content bla-bla
  </p>
  <a
    href="www.url.com"
  >
    View resource
  </a>
</div>
`;
